Server/mods/deathmatch/logic/CMainConfig.h

```diff
@@ -87,6 +87,7 @@ class CMainConfig : public CXMLConfig
     unsigned int                GetScriptDebugLogLevel() { return m_uiScriptDebugLogLevel; };
     const std::string&          GetAccessControlListFile() { return m_strAccessControlListFile; };
     bool                        GetSerialVerificationEnabled() { return m_bVerifySerials; };
+    std::map<SString, SString>  GetRulesForASE() { return m_RulesForASEMap; };
```

Comment:
`const std::map<SString, SString>& GetRulesForASE() const noexcept { return m_RulesForASEMap; }`

Reply:
Thanks for the reviews tederis